kpconv_torch/utils/config.py

```diff
@@ -16,368 +15,29 @@ class BColors(Enum):
     UNDERLINE = "\033[4m"
 
 
-class Config:
```

Comment:
By reading the diff, I realize that there is a misunderstanding between our comprehension of the config problem.

You choose to drop the `Config` class along with the dataset specific config classes. In my opinion, the dataset specific config classes should indeed by replaced with `.yml` files. However the main `Config` class is still useful to receive every config values and keep an object structure. (Bonus: this should have kept the diff reasonable.)

Reply:
I think the split between the mother and the son classes has been done only following inheritance logic. That is to say, everything that is common to all datasets is in (in principle) in the mother class and the rest in the son classes. The parameters do not play a different role, depending if they are in the mother or in the son classes. That is why I decided to group them in a unique dictionary, which appears to me to be clearer than having to mechanism to query to access the parameters.